ElectronClient/app.js

```diff
@@ -622,7 +622,7 @@ class Application extends BaseApplication {
 
 		// the following menu items will be available for all OS under Tools
 		const toolsItemsAll = [{
-			label: _('Resources'),
+			label: _('Note attachments...'),
```

Comment:
It would be better to move the ... outside of the l8n block. This way it won't need to be translated for each language twice.

Reply:
In general we indeed try to reduce the number of translations, but in this case it would be best to leave as it was originally. This is because different languages write ellipsis differently - "..." in English, but "......" in Chinese, and I've seen "。。。" too.